docs/tasks/manage-gpus/scheduling-gpus.md

```diff
@@ -6,48 +6,80 @@ title: Schedule GPUs
 
 {% capture overview %}
 
-Kubernetes includes **experimental** support for managing NVIDIA GPUs spread across nodes.
-This page describes how users can consume GPUs and the current limitations.
+{% include feature-state-alpha.md %}
+
+Kubernetes includes **experimental** support for managing NVIDIA GPUs spread across nodes using the device plugins alpha feature.
+This page describes NVIDIA's supported method on how users can consume GPUs as well as the current limitations.
+
+Starting in version 1.8, Kubernetes provides a [device plugin framework](https://github.com/kubernetes/community/blob/master/contributors/design-proposals/resource-management/device-plugin.md)
+for vendors to advertise their resources to the kubelet without changing Kubernetes core code.
+
+Instead of writing custom Kubernetes code, vendors can implement a device plugin that can be deployed manually or as a DaemonSet.
+The targeted devices include GPUs, High-performance NICs, FPGAs, InfiniBand, and other similar computing resources that may require vendor specific initialization and setup.
 
 {% endcapture %}
 
 {% capture prerequisites %}
 
-1. Kubernetes nodes have to be pre-installed with Nvidia drivers. Kubelet will not detect Nvidia GPUs otherwise. Try to re-install Nvidia drivers if kubelet fails to expose Nvidia GPUs as part of Node Capacity. After installing the driver, run `nvidia-docker-plugin` to confirm that all drivers have been loaded.
-2. A special **alpha** feature gate `Accelerators` has to be set to true across the system: `--feature-gates="Accelerators=true"`.
```

Comment:
This old feature is still in, should we keep the both by describing them in different sections?

Reply:
We intend to deprecate this feature in 1.10 is the device plugin graduates.
I don't think we want people to keep using this feature anyways.

Reply:
I think removing these two bullets is okay if we add a note:

```
**Note**: The previous Alpha support for GPU was gated by `Accelerators`. This is being deprecated in preference of the device plugin solution.
{: .note}
```